spec/requests/myjobs_pages_spec.rb

```diff
@@ -0,0 +1,23 @@
+require "rails_helper"
+
+RSpec.describe "MyjobsPages", type: :request do
+  describe "GET /myjobs_pages" do
+    before do
+      @user = create(:user)
+      allow_any_instance_of(ApplicationController)
+      .to receive(:current_user).and_return(@user)
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 0) spaces for indenting an expression spanning multiple lines.